app/setting.tsx

```
@@ -1,65 +1,151 @@
+    }
+
+    SWITCH_CONFIG[type].value.value = !SWITCH_CONFIG[type].value.value;
+  };
+
+  useEffect(() => {
+    const updateGrantedNotifications = async (
+      userId: string,
+      grantedNotifications: NotificationType[],
+    ) => {
+      await updatePushToken({ userId, grantedNotifications });
+      queryClient.invalidateQueries({ queryKey: ["pushToken", userId] });
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: SWITCH_CONFIG

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)